R/calcGDP.R

```diff
@@ -82,7 +82,7 @@ calcGDP <- function(scenario = c("SSPs", "SDPs", "SSP2EU"),
   }
 
   # Convert to US$MER if required
-  if (grepl("US$MER", unit)) {
+  if (grepl("US\\$MER", unit)) {
```

Comment:
Note: there is also the `grepl("US$MER", unit, fixed = TRUE)` option, which I find sometimes easier compared to escaping special characters.